### ISSUE TITLE
LaTeX not escaped in UDR names

### ISSUE THREAD

**Issue by harald-lang** [harald-lang/ira#16]
e.g. saving a relation with the name "s_h_v" results in malformed rendering

**Comment by harald-lang** [comment]
"s\_h\_v" works

**Comment by mathiasquintero** [comment]
you're right, it doesn't seem to work... MathJax can't compile a double subscript... We have to be able to catch the error to mitigate the problem

**Comment by harald-lang** [comment]
... or we print relation names in text-mode instead of math-mode. maybe we simply can encapsulate the names in a \textrm{...}, \text{...} or \mbox{...}

**Comment by mathiasquintero** [comment]
I still think it would be nice to be able to name relations with latex notation. And we resort to text mode when the names don't compile...
